<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myMainWindow</class>
 <widget class="QMainWindow" name="myMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="abView" name="imgShowWin" native="true"/>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>与MES通信文件夹路径:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="resultToMESPathLab">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="setResPathToMESBt">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>单个PCBID：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="onePCBID">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>区域ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="viewIDComBox"/>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>报警区域轮廓显示线宽</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="widthErrCon">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="currentText">
             <string>1</string>
            </property>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>通信计算机IP</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="toConnectIP">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="inputMask">
             <string>000.000.000.000</string>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::VisualMoveStyle</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="revokeBt">
            <property name="text">
             <string>撤销修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveBt">
            <property name="text">
             <string>保存修改</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTableView" name="carrierAutoResTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="carrierManuResTableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menuOpen_database">
     <property name="title">
      <string>打开数据库</string>
     </property>
     <addaction name="actionSetARdb"/>
     <addaction name="actionOpenMRdb"/>
    </widget>
    <addaction name="actionNewDB"/>
    <addaction name="menuOpen_database"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menuSetModel">
     <property name="title">
      <string>设置复检模式</string>
     </property>
     <addaction name="actionOKAuto"/>
     <addaction name="actionOKNotAuto"/>
    </widget>
    <addaction name="actionSetPathToMES"/>
    <addaction name="actionSetResFilePath"/>
    <addaction name="menuSetModel"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actLogin"/>
   <addaction name="separator"/>
   <addaction name="actSettings"/>
   <addaction name="actViewDatabase"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actLogin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources/iconsHuman.qrc">
     <normaloff>:/images/images/login.png</normaloff>:/images/images/login.png</iconset>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>Login/Logout</string>
   </property>
  </action>
  <action name="actionSetPathToMES">
   <property name="text">
    <string>设置MES通信文件夹路径</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="icon">
    <iconset resource="Resources/iconsHuman.qrc">
     <normaloff>:/images/images/setting.png</normaloff>:/images/images/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>Setting path and model</string>
   </property>
  </action>
  <action name="actViewDatabase">
   <property name="icon">
    <iconset resource="Resources/iconsHuman.qrc">
     <normaloff>:/images/images/view_database.png</normaloff>:/images/images/view_database.png</iconset>
   </property>
   <property name="text">
    <string>数据库视图</string>
   </property>
   <property name="toolTip">
    <string>view database</string>
   </property>
  </action>
  <action name="actionNewDB">
   <property name="text">
    <string>新建人工复检数据库</string>
   </property>
  </action>
  <action name="actionOKAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>复检自动检测为合格的产品</string>
   </property>
  </action>
  <action name="actionOKNotAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>不复检自动检测为合格的产品</string>
   </property>
  </action>
  <action name="actionSetResFilePath">
   <property name="text">
    <string>设置TXT文件存储路径</string>
   </property>
  </action>
  <action name="actionSetARdb">
   <property name="text">
    <string>设置自动检测数据库文件</string>
   </property>
  </action>
  <action name="actionOpenMRdb">
   <property name="text">
    <string>打开人工附件结果数据库</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>abView</class>
   <extends>QWidget</extends>
   <header>abview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources/iconsHuman.qrc"/>
 </resources>
 <connections/>
</ui>
